src/victory-util/add-events.js

```diff
@@ -0,0 +1,61 @@
+import { defaults, assign, isFunction, partialRight } from "lodash";
+import Events from "./events";
+
+export default (WrappedComponent) => {
+  return class addEvents extends WrappedComponent {
+    constructor() {
+      super();
+      this.state = {};
+      const getScopedEvents = Events.getScopedEvents.bind(this);
+      this.getEvents = partialRight(Events.getEvents.bind(this), getScopedEvents);
+      this.getEventState = Events.getEventState.bind(this);
+    }
+
+    componentWillMount() {
```

Comment:
it's just that since this is meant to be used by other future components it seems a good idea to at least warn when overwriting methods (eg. victory-force should need at least a `componentWillMount` to set up the `simulation`)